### ISSUE TITLE
Debugger port conflict when you fork children

### ISSUE THREAD

**Issue by FranzZemen** [naptha/tesseract.js#66]
I'm debugging my app and get a EADDRINUSE error when Tesseract.recognize is called.   I hunted this down and I think its because you don't specify execArgv.  See the following issue others have with debugging and forking in general:

https://github.com/nodejs/node/issues/3469

As it stands, I can't debug to inspect the result.

**Comment by ilja903** [comment]
Same issue, very annoying that debugger is not working. Any hacks?

**Comment by ztz0223** [comment]
I saw this issue too, looks like conflict with the nodemon

**Comment by tetreault** [comment]
bump

**Comment by seitbekir-seidametov-codeit** [comment]
Not really good solution, but OK fr debugging.

```
    const old = process.execArgv[0]
    process.execArgv.length = 0

    Tesseract.recognize(imagination)
```

**Comment by jeromewu** [comment]
Close for now, please check v2 if the issue exists and reopen if needed.

**Comment by aeisenberg** [comment]
Please re-open this bug. It is still a problem in v2.0.0-beta.1.

Run this file with `node --inspect`:
```
const Tesseract = require('tesseract.js');
(async () => {
  await Tesseract.recognize('', 'eng', { logger: m => console.log(m) });
})();
```

**Comment by jeromewu** [comment]
Fixed in 2.0.0-beta.2, please check.
